INSTALL.md

```diff
@@ -14,6 +14,8 @@ run:
 
 ## Prerequisites
 
+The first step is to install `bazel` using the instructions found on the [`bazel`][bazel][^1] site.
```

Comment:
Can you change `[bazel][bazel]` to `[bazel][bazel-install]` and add `[bazel-install]: https://bazel.build/install` below `[bazel-config]: https://bazel.build/run/bazelrc` at the bottom of the file?